docs/how-to-use-kit.md

```diff
@@ -0,0 +1,105 @@
+
+* `Create a KIT guest/EKS cluster`
+* `Create n number of nodes` or
+* `Create x number of Kubernetes objects.`
+
+A user can combine these various tasks to run these tests in a pipeline and collect results. 
+
+If you are starting with KIT refer [use-cases covered by KIT](#use-cases-supported-with-kit) to make sure your use-case is supported, if its not listed or its not clear feel free to file an issue in the repo.
+
+## Getting started
+
+* `kitctl` binary helps with creating the KIT environment in an AWS account. Install kitctl locally on your mac-
+
+```bash
+brew tap awslabs/kit https://github.com/awslabs/kubernetes-iteration-toolkit.git
+brew install kitctl 
+```
+
+* Bootstrap a KIT environment using the kitctl
+
+```bash
+export AWS_REGION=us-west-2
+kitctl bootstrap kitctl-$(whoami) # Optional environment name
+```
+
+Note: Make sure you are logged into the AWS account and have admin privilege, and the environment name is unique to you as KIT uses it create an S3 bucket in your AWS account.
+
+* Bootstrap takes about 5 minutes,  as part of this bootstrap it configures the AWS infra such as VPC, security groups, subnets, route tables etc. and a management Kubernetes cluster. At the end of the bootstrap command it prints an `export KUBECONFIG` command, run this command to access the management cluster using kubectl.
+
+Note: This environment we just created comes pre-installed with Tekton for running the tests and a Prometheus stack for monitoring.
+
+* To access the Tekton dashboard `kubectl port-forward svc/tekton-dashboard -n tekton-pipelines 9097:9097` and you will see the templates for tasks and pipelines in Tekton. If you need more info about Tekton terminology <refer here> TODO
+* To access the Grafana dashboard `kubectl port-forward svc/kube-prometheus-stack-grafana -n monitoring 8080:80` <Add more details about the dashboards> TODO
+
+## Running tests
+
+At this point a user can run tests against either an EKS cluster or vanilla Kubernetes cluster provisioned by KIT operator called [guest clusters](https://quip-amazon.com/xCtbA6C6X7dy/How-to-use-KIT-for-Kubernetes-testing#temp:C:VfOa1f3b6a5d7de51afb05d574ef) running in the environment. 
+Difference between testing against the two cluster types-
+
+* EKS cluster doesn’t allow changing any flags or control plane images, however, if you want to make any flags changes or want to run custom docker images for API server or any other Kubernetes control plane component use guest cluster for testing
+* Metrics that we collect from EKS clusters are limited to API server metrics and CW metrics, we don’t get the node level metrics from EKS control plane nodes. In guest clusters, we are able to collect API server, etcd, scheduler, KCM and node level metrics.
```

Comment:
Should we mention that CW metrics are available given only internal teams will have these?